torch/nn/quantized/modules/_generated.py

```diff
@@ -0,0 +1,27 @@
+
+# @generated This file is produced by `torch/quantization/tools/make_module`.
+
+import torch
+
+r"""Add wraps the torch.ops.quantized.add function."""
+class Add(torch.nn.Module):
+    __FLOAT_MODULE = torch.nn.modules.Add
+
+    def __init__(self, scale=1.0, zero_point=0):
+        super(Add, self).__init__()
+        self.scale = 1.0
```

Comment:
I think these are supposed to be tensors?

Reply:
You need to assign them from init args

Reply:
No, they cannot be tensors, because the ops expect double and long. However, if saving it as `buffer`, we might need to convert them to tensors, and use `.item()` when calling the op